docs/conf/proxy.md

```diff
@@ -31,6 +31,16 @@ import sttp.client3._
 SttpBackendOptions.httpProxy("some.host", 8080, "username", "password")
 ```
 
+If your backend implements `HttpClientBackend` and your proxy server requires Basic authentication, you will need to enable it by 
```

Comment:
minor correction: if you use a `HttpClient`-based backend (e.g. `HttpClientBackend`)

that is, backends are used, not implemented :)